addons/website/tests/test_menu.py

```diff
@@ -185,3 +189,35 @@ def test_02_menu_anchors(self):
         self.assertFalse(self.menu.page_id, "M2o should have been unset as this is an anchor URL.")
         self.assertEqual(self.menu.url, self.page_url + '#anchor', "Page URL should have been properly prefixed with the referer url")
         self.assertEqual(self.page.url, self.page_url, "Page URL should not have changed")
+
+    def test_03_mega_menu_translate(self):
+        # Setup
+        fr = self.env['res.lang']._activate_lang('fr_FR')
+        Menu = self.env['website.menu']
+        website = self.env['website'].browse(1)
+        website.language_ids = [Command.link(fr.id)]
+        menu = Menu.create({
+            'name': 'Test Mega Menu Content Translation Edit Mode',
+            'mega_menu_content': '<p>something</p>',
+            'parent_id': website.menu_id.id,
+            'website_id': website.id,
+        })
+        self.env['ir.module.module']._load_module_terms(['website'], [fr.code])
+
+        # Load cache
+        self.url_open('/%s' % fr.url_code)
+        self.url_open('/%s?edit_translations=1' % fr.url_code)
+
+        # Translate
+        root = html.fromstring(menu.mega_menu_content, parser=html.HTMLParser(encoding="utf-8"))
```

Comment:
Hi, @rdeodoo 
could you help me check if the `parser=html.HTMLParser(encoding="utf-8")` is still necessary here?

Reply:
It's useless.
I copy pasted this line when I wrote this part of the test (with the `sha256` below).
The default `parser` used in `lxml` is already the `HTMLParser` if omitted.

Also, about the encoding, it's probably useless, from what I know in python3, utf-8 is generally the default in most libs.